lib/contracts.ncl

```diff
@@ -45,6 +44,11 @@ in
   # instead?
   System = { arch | String, os | String },
 
+  NullOr
```

Comment:
Ok, we REALLY need this to be a builtin type. You shouldn't have to define that by yourself in 1.0.0+